README.md

```diff
@@ -38,21 +35,18 @@ that are too broadly scoped to lead to agreement, so be as specific as you can.
 
 **Example:** *The Barricade can be circumvented too easily with wall jumps.*
 
-## Proposal
+## Idea, that solves a problem
 
-You may propose a change addressing **an agreed upon opinion**, which you must
-cite within your issue. We will only discuss your proposal, not the opinion,
+You may propose a change addressing **at least one problem issue**, which you must
+cite within your issue. If at least one of the cited problems was accepted we will only discuss your idea, not the problem,
```

Comment:
I'm fine with the change to allow solution ideas already for fresh perceived problems but I find your wording here too vague; it is unclear what happens if no problem was agreed upon initially.

Reply:
an idea without an agreed upon problem would be closed after 30 days

Reply:
The problem is that the claim "we will discuss your idea but not the problem it addresses" is now linked to your new "if" clause while previously it was an uncodonditional fact:

> **If** at least one of the cited problems was accepted **[then]** we will only discuss your idea, not the problem, […]

I'm sure this is not what you meant to express with that change.

Reply:
Oh, yes now i understand it. 😄